packages/connect-core/src/utils/index.ts

```diff
@@ -0,0 +1,4 @@
+export { findAppMethodFromIntent } from './app'
```

Comment:
I decided to expose this utilities, what you think?

Reply:
Sounds good yes!